test/e2e/tests/onboarding.spec.js

```diff
@@ -19,6 +28,218 @@ describe('MetaMask onboarding', function () {
       },
     ],
   };
+
+  it('Clicks create a new wallet, accepts a secure password, reveals the Secret Recovery Phrase, confirm SRP', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await completeCreateNewWalletOnboardingFlow(
+          driver,
+          testSeedPhrase,
+          testPassword,
+        );
+
+        const homePage = await driver.findElement('.home__main-view');
+        const homePageDisplayed = await homePage.isDisplayed();
+
+        assert.equal(homePageDisplayed, true);
+      },
+    );
+  });
+
+  it('Clicks import a new wallet, accepts a secure password, reveals the Secret Recovery Phrase, confirm SRP', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await completeImportSRPOnboardingFlow(
+          driver,
+          testSeedPhrase,
+          testPassword,
+        );
+
+        const homePage = await driver.findElement('.home__main-view');
+        const homePageDisplayed = await homePage.isDisplayed();
+
+        assert.equal(homePageDisplayed, true);
+      },
+    );
+  });
+
+  it('User import wrong Secret Recovery Phrase', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder({ onboarding: true }).build(),
+        ganacheOptions,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+
+        await importWrongSRPOnboardingFlow(driver, wrongSeedPhrase);
+
+        const confirmSeedPhrase = await driver.findElement(
+          '[data-testid="import-srp-confirm"]',
+        );
+
+        assert.equal(await confirmSeedPhrase.isEnabled(), false);
+      },
+    );
+  });
+
+  it('Check if user select different type your secret recovery phrase', async function () {
```

Comment:
I think here you mean _Check if user select different type **of** secret recovery phrase_

Reply:
Done, my bad